google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1509,7 +1509,7 @@ public StorageClass getStorageClass() {
    *     (CORS)</a>
    */
   public List<Cors> getCors() {
-    return cors;
+    return cors != null ? cors : ImmutableList.<Cors>of();
```

Comment:
Please leave this field as is and revert change. There are potentially 4 cases which are considered here and we don't want to mess them up.

1. Cors are set and field selector is selected then returns not-null.
2. Cors are set but field selector isn't selected then returns null.
3. Cors are not set and field selector is selected then returns null.
4. Cors are not set and field selector isn't selected then returns null.

Reply:
Done